<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseCfgDlg</class>
 <widget class="QDialog" name="DatabaseCfgDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>269</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>269</width>
    <height>269</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>269</width>
    <height>269</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>数据库配置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="driverLabel">
     <property name="text">
      <string>数据库类型：</string>
     </property>
     <property name="buddy">
      <cstring>driverComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="driverComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <item>
      <property name="text">
       <string>QSQLITE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QMYSQL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QMYSQL3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QODBC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QODBC3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QPSQL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QPSQL7</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="hostLabel">
     <property name="text">
      <string>主机地址：</string>
     </property>
     <property name="buddy">
      <cstring>hostLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="hostLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>localhost</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="dbNameLabel">
     <property name="text">
      <string>数据库名：</string>
     </property>
     <property name="buddy">
      <cstring>dbLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="dbLineEdit">
     <property name="text">
      <string>testdb</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="portLabel">
     <property name="text">
      <string>端口号：</string>
     </property>
     <property name="buddy">
      <cstring>portSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="portSpinBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>3306</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="userLabel">
     <property name="text">
      <string>用户名：</string>
     </property>
     <property name="buddy">
      <cstring>userLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="userLineEdit">
     <property name="text">
      <string>testuser</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="pwdLabel">
     <property name="text">
      <string>密码：</string>
     </property>
     <property name="buddy">
      <cstring>pwdLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="pwdLineEdit">
     <property name="text">
      <string>431134</string>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>driverComboBox</tabstop>
  <tabstop>hostLineEdit</tabstop>
  <tabstop>dbLineEdit</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>pwdLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
